test/js/parsingTest.js

```diff
@@ -89,4 +89,31 @@ describe('parsing', function () {
       })
     })
   })
+  describe('regex parsing', function () {
+    it('not a regex should not be considered a regex', function (cb) {
```

Comment:
not to be silly, but this wont make sense to most people who aren't already familiar with the problem.  A different test description would be more useful to others reading the code

Reply:
@snyderp I updated with a comment to see the issue for details.

Reply:
![image](https://user-images.githubusercontent.com/76526/52391679-5ba98800-2a53-11e9-8849-3110d4120280.png)

Reply:
@snyderp I also updated the description to "regex should not check last character of input buffer"